api/user_role/schema.py

```diff
@@ -19,6 +20,10 @@ class Arguments:
     user_role = graphene.Field(UsersRole)
 
     def mutate(self, info, **kwargs):
+        roles = UserRoleModel.query.filter_by(user_id=kwargs['user_id']).\
```

Comment:
@migot01 I would suggest you collapse the two `filter_by()` into one and also in the if condition I think `len` is not necessary  as simple `if roles:` is better.